cnn_framework/uncertainty_test.py

```diff
@@ -27,3 +27,23 @@ def testEnsembleModel(self):
         expected_std_output = np.array([[0.5,0,0.5],[0.5,0,0.5],[0,0.5,0.5]])
         self.assertTrue(np.array_equal(y_output[0],expected_y_output))
         self.assertTrue(np.array_equal(std_output[0],expected_std_output))
+    
+    def testEnsembleTrain(self):
```

Comment:
Squash this part with the other unit test commit for `uncertainty_test.py`. Maybe you could squash this entire commit with that one. Up to you.

Reply:
Done